Mathlib/Algebra/Order/Floor.lean

```diff
@@ -1736,21 +1736,21 @@ theorem Nat.ceil_int : (Nat.ceil : ℤ → ℕ) = Int.toNat :=
 
 variable {a : α}
 
-theorem Nat.cast_floor_eq_int_floor (ha : 0 ≤ a) : (⌊a⌋₊ : ℤ) = ⌊a⌋ := by
+theorem Int.ofNat_floor_eq_floor (ha : 0 ≤ a) : (⌊a⌋₊ : ℤ) = ⌊a⌋ := by
```

Comment:
It uses `Nat.cast`, not `Int.ofNat` (do we have a naming convention about this?). Also, please use `@[deprecated]` and/or list all renames in the commit message.

Reply:
It does use `Int.ofNat` under the hood. I would usually use `Nat.cast` in the name, but I want to make an exception here because I have been really struggling to find this lemma in the past and I think it's better to explicitly mention `Int.ofNat` in the name to avoid confusion over what `Nat.cast` is casting to.

I think there's no point using `deprecated` when nothing downstream of the file broke (that is, I believe this is a good criterion to decide whether a lemma is popular or not). See https://leanprover.zulipchat.com/#narrow/stream/345428-mathlib-reviewers/topic/Mathlib.20deprecation.20cycle

Reply:
As you can see in that discussion (note to public: it's a discussion in a closed Zulip stream), at least @digama0, @semorrison, and @kbuzzard disagree with you. So, please add `@[deprecated] alias`es (it's not that hard for 4 lemmas), then I'll merge this PR.